doc/script_commands.txt

```diff
@@ -11132,6 +11132,13 @@ Returns 1 on success.
 
 ---------------------------------------
 
+*channel_join "<chname>"{, <char_id>};
+
+Join an existing public or private channel.
```

Comment:
I dont really see a check for `CHAN_TYPE_*`
Maybe it is not really necessary, but just wanted to point it out.

Reply:
I think its not really necessary.